libtbx/auto_build/bootstrap.py

```diff
@@ -2653,9 +2672,16 @@ def run(root=None):
                     help="Builds software with mpi functionality",
                     action="store_true",
                     default=False)
-  parser.add_argument("--python3",
+  python_args = parser.add_mutually_exclusive_group(required=False)
+  python_args.add_argument('--python',
+                    default='27', type=str, nargs='?', const='27',
+                    choices=['27', '36', '37', '38'],
+                    help="""When set, a specific Python version of the
+conda environment will be used. This only affects environments selected with
+the --builder flag. This cannot be used with --python3.""")
+  python_args.add_argument("--python3",
                     dest="python3",
-                    help="Install a Python3 interpreter. This is unsupported and purely for development purposes.",
+                    help="Install a Python3 interpreter. This is unsupported and purely for development purposes. This cannot be used with --python.",
```

Comment:
```suggestion
                    help=argparse.SUPPRESS,
```
I don't think there is a need to advertise this option any more. The help string is incorrect in any case (it is supported).
I think this option can either be hidden or even be removed entirely.

Reply:
Sure, I can remove the option. How about the wxpython4 one?

Reply:
wxpython4 is implied by python 3 environments, so the flag is not needed there.

However the flag is also automatically set on python 2 when running on MacOS 10.14+.
Removing the option entirely may break those platforms. (I'm fine with it being removed as we no longer support those cases on master)

Reply:
having said that - the command line option of course is not needed. So we could instead move the MacOS check to the point where the wxpython 3 or 4 decision is made.